controllers/pullRequestsController.js

```diff
@@ -107,7 +107,12 @@ const getStalePRs = async (req, res) => {
  */
 const getOpenPRs = async (req, res) => {
   try {
-    const { data } = await githubService.fetchOpenPRs()
+    let pageNumber = 1
+
+    if (req.query.page > 1) {
+      pageNumber = req.query.page
+    }
+    const { data } = await githubService.fetchOpenPRs(pageNumber)
```

Comment:
There is a better and short way to pass default arguments

Please have a look here:
https://github.com/Real-Dev-Squad/website-backend/blob/d17ea626918d1424cc8ebd8181794c572e96ac24/models/users.js#L60

Also, please make sure to convert the argument to integer using `parseInt` as done in above function.